.gitattributes

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Justin Myers for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
+
+* text eol=lf
```

Comment:
Same as the other PR. I think this is too broad and should be limited to some file extensions in this repo.

Reply:
The latest commit changes this to use a more specific list of extensions.